ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/events/EventApplication.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.events;
+
+import static spark.Spark.get;
+
+import java.io.PrintWriter;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.servlet.SparkApplication;
+
+public class EventApplication implements SparkApplication {
```

Comment:
I didn't look into it, but it looks like I could switch it over to Javalin if that's preferred.

Reply:
Javalin is the direction we would like to go since it supports using it as a servlet out of the box unlike Spark.  It is also nice to use their implementation since it has probably be more widely used and tested than our current version.